core/trace_point.rbs

```diff
@@ -421,7 +421,7 @@ class TracePoint < Object
   # Value from exception raised on the `:raise` event, or rescued on the `:rescue`
   # event.
   #
-  def raised_exception: () -> untyped
+  def raised_exception: () -> Exception
```

Comment:
How is the case of other events than `:raise` nor `:rescue`?

```suggestion
  def raised_exception: () -> Exception?
```

Reply:
If you call `raised_exception` and there's no exception, it raises an error:
```
$ ruby -e 'TracePoint.trace(:line) { p _1.raised_exception }; p 1'
<internal:trace_point>:402:in `raised_exception': not supported by this event (RuntimeError)
	from -e:1:in `block in <main>'
	from -e:1:in `<main>'
```